WebAPIClient/WebAPIClient/Program.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Threading.Tasks;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text.Json;
+using System.Collections.Generic;
+
+namespace WebAPIClient
+{
+    class Program
+    {
+        /// <summary>
+        /// Create instance of HttpClient for use accessing .NET Foundation repos via GitHub API
+        /// </summary>
+        private static readonly HttpClient client = new HttpClient();
```

Comment:
Noter here the HttpClient is static so a single instance will be used across the application. It takes a good deal of resources for a HttpClient so its useful to run one across a client app.